src/packages/demon.demoncord.json

```diff
@@ -1,8 +1,8 @@
 {
 	"name": "Demoncord Loader",
 	"id": "demon.demoncord",
-	"description": "A Discord client mod by satanists, for satanists",
-	"tags": ["discord", "demoncord", "satanism"],
+	"description": "A Discord client mod by satanists, for satanists.",
+	"tags": ["Discord", "Demoncord", "Satanism"],
```

Comment:
why are they capitalized; literally no other tags in the entire repo are capitalized

Reply:
They were but Tropical reverted it.

Reply:
fair enough